src/json.rs

```diff
@@ -484,10 +484,8 @@ fn flatten_dbus_stats(
                 if let Some(val) = value {
                     flat_stats.insert(
                         format!("{base_metric_name}.user.{user_name}.{field_name}"),
-                        val.cur.into(),
+                        val.get_usage().into(),
```

Comment:
Also, I'll have to use this in fbmonitord 

I might go and refactor this to only store this value and ditch keeping cur and max in memory ... just to save memory and avoid confusion? Thoughts?

But happy to stay with your store all the data and display / use only what we want, just memory usage is all I worry about as being a WDB using as little as possible is a goal.